src/sol_imgui/sol_imgui.h

```diff
@@ -51,6 +60,18 @@ inline bool BeginChild(const std::string& name, float sizeX, float sizeY, bool b
 {
     return ImGui::BeginChild(name.c_str(), {sizeX, sizeY}, border, flags);
 }
+
+// new
+inline bool BeginChild(const std::string& name, float sizeX, float sizeY, int child_flags)
+{
+    return ImGui::BeginChild(name.c_str(), {sizeX, sizeY}, static_cast<ImGuiChildFlags>(child_flags));
+}
+
+inline bool BeginChild(const std::string& name, float sizeX, float sizeY, int child_flags, int window_flags)
```

Comment:
please, use int32_t inatead of int

Reply:
the entire rest of the header uses `int` rather than `int32_t`. should i update all of them?

Reply:
i guess you may bulk change them

Reply:
actually lets not do this in this PR